### PR TITLE
2.0 docs project structure

### PR DESCRIPTION
**Pull request by maasg**
This PR adds documentation to describe the project structure that Cloudflow applications require for single and multi-runtime applications.

### CODE REVIEW

File: docs/shared-content-source/docs/modules/develop/pages/project-structure.adoc [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+│   ├── build.properties      #<3>
+│   ├── cloudflow-plugins.sbt #<4>
+│   └── plugins.sbt           #<5>
+├── src                       #<6>
+│   └── main
+└── target-env.sbt            #<7>
+----
+<1> The `build.sbt` file specifies the build configuration, project dependencies and eventual sub-projects.
+<2> The `project\` folder contains additional configuration for the project build, like plugins and sbt versions. This folder is mandatory.
+<3> `build.properties` file contains the `sbt` version
```
Comment (RayRoestenburg) [inline docs/shared-content-source/docs/modules/develop/pages/project-structure.adoc:27]:
```suggestion
<3> The `build.properties` file contains the `sbt` version 
```

**Comment by maasg** [comment]
closes #514

File: docs/shared-content-source/docs/modules/develop/pages/project-structure.adoc [hunk 2]
```diff
@@ -0,0 +1,164 @@
+:page-partial:
+
+include::ROOT:partial$include.adoc[]
+
+Cloudflow uses and extends `sbt`, the _scala build tool_, to support the building, packaging, and local execution of applications.
+The project structure of a Cloudflow application follows the general conventions and folder structure of a `sbt` project with a few additional considerations.
```
Comment (yuchaoran2011) [inline docs/shared-content-source/docs/modules/develop/pages/project-structure.adoc:6]:
```suggestion
The project structure of a Cloudflow application follows the general conventions and folder structure of an `sbt` project with a few additional considerations.
```

File: docs/shared-content-source/docs/modules/develop/pages/project-structure.adoc [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+<1> use `enablePlugins` to activate the Cloudflow plugins for this project.
+
+NOTE: See the complete example in the https://github.com/lightbend/cloudflow/blob/{cloudflow-branch-version}/examples/snippets/modules/ROOT/examples/akkastreams-scala/build.sbt[examples folder of the Cloudflow project] 
+
+We provide templates to get started with a Cloudflow project that uses a single runtime. See
+https://github.com/lightbend/cloudflow/blob/{cloudflow-branch-version}/examples/templates[templates folder of the Cloudflow project]
+
+== Multiple Runtime Applications
+
+Applications that combine multiple streamlet runtimes *must be* organized using `sbt` https://www.scala-sbt.org/1.x/docs/Multi-Project.html[multi-projects builds].
```
[the remark this answers is not in the document]
Comment (maasg) [reply docs/shared-content-source/docs/modules/develop/pages/project-structure.adoc:113]: :heavy_check_mark:

**Comment by maasg** [comment]
Also merged manually into `v2.0.0-docs`

**Pull request merged**